### DISCUSSION TITLE
Help T_T

### DISCUSSION THREAD

**Discussion by Marco-9456** [JunkFood02/Seal#401] [Q&A] [answered]
So I'm downloading a Crunchyroll series called JUJUTSU KAISEN, but I have an issue with its episode names, 
I want to make the episode names look like this: `Show name - S01E01 - episode title` but I keep failing 😩, I used this template 
```
--downloader libaria2c.so
--external-downloader-args "aria2c:\"--summary-interval=120\""
--embed-thumbnail
--convert-thumbnails png
--remux-video mkv
--sub-langs ar-ME
--write-sub
--embed-subs
--compat-option no-keep-subs
-o "/storage/emulated/0/Download/Seal/Video/Crunchyroll/%(series)S/%(season_number)s - %(series)S - S%(season_number)s E%(episode_number)s - %(title).%(ext)s"
```

I would appreciate any suggestions to get the result that I need 🙏.

**Comment by JunkFood02** [comment, marked as answer]
How does this command "failing"?
`--write-sub` `--compat-option no-keep-subs` and the path in `-o` looks redundant since Seal would add `-P` 

try `-o "Crunchyroll/%(series)S/%(season_number)s - %(series)S - S%(season_number)s E%(episode_number)s - %(title).%(ext)s"`

**Reply by Marco-9456** [reply]
I'm getting this now 

```
Usage: yt-dlp [OPTIONS] URL [URL...]

yt-dlp: error: invalid default output template "Crunchyroll/%(series)S/%(season_number)s - %(series)S - S%(season_number)s E%(episode_number)s - %(title).%(ext)s": unsupported format character '%' (0x25) at index 106

```

**Reply by JunkFood02** [reply]
You forgot an `s` after the title :rofl:

**Reply by Marco-9456** [reply]
> You forgot an `s` after the title 🤣

Oh 🤣🤣 my bad its working now.
